snappy/binaries.go

```diff
@@ -143,10 +138,7 @@ func addPackageBinaries(m *snapYaml, baseDir string) error {
 			continue
 		}
 
-		aaProfile, err := getSecurityProfile(m, app.Name, baseDir)
-		if err != nil {
```

Comment:
Not sure if the context is okay but this cuts away the ``if err != nil { ... }`` code.